package.json

```diff
@@ -37,8 +37,8 @@
   },
   "devDependencies": {
     "@atomist/automation-client": "^1.4.0",
-    "@atomist/sdm": "^1.4.1",
-    "@atomist/sdm-core": "^1.4.0",
+    "@atomist/sdm": "^1.5.0-remove-deployer.20190430085219",
+    "@atomist/sdm-core": "^1.5.0-remove-deployer.20190430091923",
```

Comment:
I wouldn’t recommend depending on those versions. We won’t merge is in the pre 2.0 stream.

Reply:
Corrected.